### DISCUSSION TITLE
Action buttons in Item List Page and Item Detail Page

### DISCUSSION THREAD

**Discussion by VaZark** [directus/directus#11849] [Feature Requests]
Usecase:
Sometimes we need to trigger some logic that has to be executed that lead to a change in status and other values and cannot be invoked with webhook which require a change to trigger

Proposal: 
Create a new type of extension that allows devs to add buttons in the Item List page and the Item Detail page that calls custom code attached in the extension.

Reference:
![image](https://user-images.githubusercontent.com/11616688/155570399-3c58746f-4aec-4d8b-8bd6-ae0acfc8c2a7.png)

**Comment by rijkvanzanten** [comment]
Related to https://github.com/directus/directus/discussions/3841. The idea there is basically to have a new "slots" based extension type, that can render in one of many "slots". A custom column in the tabular layout could be one of these "slots" 🙂

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
